### PR TITLE
Implementation of the Xtals Webform

### PR DESCRIPTION
**Pull request by AnnaTsolakou**

### CODE REVIEW

**Comment by viis** [comment]
Hi Anna,

Great to see your first pull request.

There is a lot of unnecessary whitespace in the code (bank lines), and that makes the code more difficult to read, especially when the use of whitespace is inconsistent. Please have a look at the style guide (http://docs.gpcrdb.org/coding_style.html) and update your pull request.

**Comment by AnnaTsolakou** [comment]
Hello Vignir,

thank you for the recommendation. I will go through the code and fix the
whitespace, and make again my pull request.

Best,
Anna

2016-04-18 21:03 GMT+02:00 Vignir Isberg <notifications@github.com>:

> Hi Anna,
>
> Great to see your first pull request.
>
> There is a lot of unnecessary whitespace in the code (bank lines), and
> that makes the code more difficult to read, especially when the use of
> whitespace is inconsistent. Please have a look at the style guide (
> http://docs.gpcrdb.org/coding_style.html) and update your pull request.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/protwis/protwis/pull/24#issuecomment-211532068>
>



-- 
Anna Tsolakou
Faculty of Science
MSc Bioinformatics
Tel: +4571583112

**Comment by AnnaTsolakou** [comment]
Hi Vignir, I have recommited the code, removing the whitespace.

File: structure/forms.py [hunk 1]
```diff
@@ -0,0 +1,458 @@
...
+    name_cont = forms.CharField(label='Name', label_suffix='' , max_length=50, required=True)
+    date=forms.DateField(widget = SelectDateWidget) #or alternatively (initial=datetime.date.today)
+    pi_name = forms.CharField(label="Name of PI (group leader)", label_suffix='',max_length=50, required=True)
+    pi_email=forms.EmailField(label="PI Contact e-mail address", max_length=50,label_suffix='', required=True)  #how to add unique, invalid ?
+    url = forms.URLField(initial='http://Database/', label='Group URL',label_suffix='', required=True )
+    address = forms.CharField(label='Affiliation address', label_suffix='', widget=forms.Textarea, required=True)
+
+    
+
+#class construct_info(forms.Form):
```
Comment (viis) [inline structure/forms.py]: Is this supposed to be commented out?

File: home/templates/home/base.html [hunk 2]
```diff
@@ -10,6 +10,11 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>{% site_title %}</title>
 
+ 
+
+
+
+
```
Comment (viis) [inline home/templates/home/base.html]: There are still many empty lines, like here.

I know it's nit-picky, but we have to try to keep the code as consistent and readable as possible

**Pull request merged**